FluentScreenRecorder/Views/MainPage.xaml.cs

```diff
@@ -63,15 +65,21 @@ public sealed partial class MainPage : Page
 
         public bool filesInFolder;
 
+        private ObservableCollection<ThumbItem> thumbnailsList = new();
+
+        public static MainPage Current;
+
         public MainPage()
         {
             InitializeComponent();
+            Current = this;
+
             Loaded += LoadedHandler;
 
             SilentPlayer = new MediaPlayer() { IsLoopingEnabled = true };
             SilentPlayer.Source = MediaSource.CreateFromUri(new Uri("ms-appx:///Assets/Silence.ogg"));
             SilentPlayer.Play();
-            ApplicationView.GetForCurrentView().SetPreferredMinSize(new Size(400, 260));
+            ApplicationView.GetForCurrentView().SetPreferredMinSize(new Size(400, 86));
```

Comment:
86 is not a multiple of 4, please change it either to 84 or 88 checking it doesn't break the look when the window is small

Reply:
Oh right, thought that's a multiple of 4.